docs/src/componentDocs/DrawerNavGroup/examples/SpacingNavGroups.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {SpacingNavGroupsExample} from './SpacingNavGroupsExample';
+
+const codeSnippet = `<Drawer style={{width: 250, margin: 'auto'}}>
+    <DrawerBody>
+        <DrawerNavGroup title="Group 1" hidePadding titleDivider={false}>
+            <DrawerNavItem title="Item 1" itemID="1" />
+            <DrawerNavItem title="Item 2" itemID="2" />
+        </DrawerNavGroup>
+        <Spacer height={30} style={{flexBasis: 'auto'}} />
```

Comment:
you can remove style={{flexBasis: 'auto'}} from snippet, this is just a fix for a bug with RNWeb

Reply:
Spacer height is not reflecting without flexBasis: auto, hence added it.